src/components/App.tsx

```diff
@@ -1853,32 +1854,25 @@ class App extends React.Component<AppProps, AppState> {
     this.actionManager.executeAction(actionToggleHandTool);
   };
 
-  /**
-   * Zooms on canvas viewport center
-   */
-  zoomCanvas = (
-    /** decimal fraction between 0.1 (10% zoom) and 30 (3000% zoom) */
-    value: number,
-  ) => {
-    this.setState({
-      ...getStateForZoom(
-        {
-          viewportX: this.state.width / 2 + this.state.offsetLeft,
-          viewportY: this.state.height / 2 + this.state.offsetTop,
-          nextZoom: getNormalizedZoom(value),
-        },
-        this.state,
-      ),
-    });
-  };
-
```

Comment:
why remove this API? I wanted to expose publicly later, and also reuse it in the codebase where we zoom (actions).

Reply:
Oh, right, that makes sense. Was just a bit hasty here, I suppose. My bad, I'll put it back.

Reply:
Ookay, I kinda lied, turns out I didn't even need that value since it was already available. 🤦‍♂️ Apologies for the confusion, should be fixed now, though, and `zoomCanvas` is back.